src/app.js

```diff
@@ -1 +1,24 @@
+/* eslint-disable no-console */
 'use strict';
+
+const operation = String(process.argv.splice(2, 1));
+const nums = process.argv.slice(2).map(num => !isNaN(num)
```

Comment:
```suggestion
const nums = process.argv.slice(2).filter(num => !isNaN(Number(num)))
```Be more careful, you don't need it map :)